app/uk/gov/hmrc/hmrcemailrenderer/templates/helpers/SalutationHelper.scala

```diff
@@ -63,6 +63,17 @@ object SalutationHelper {
     }
   }
 
+  def saluteFullName(params: Map[String, Any]): String = {
```

Comment:
With this file, there is already a method 'def informalSalutationFrom' which should perform the same task.

Reply:
this new method uses recipientName_FullName which is all we are passing, the informalSalutationFrom uses recipientName_forename & recipientName_surname, so I needed a new method